WordPress/src/main/java/org/wordpress/android/datasets/ReaderLikeTable.java

```diff
@@ -77,14 +77,18 @@ public static void setCurrentUserLikesPost(ReaderPost post, boolean isLiked, lon
         if (post == null) {
             return;
         }
+        setCurrentUserLikesPost(post.postId, post.blogId, isLiked, wpComUserId);
+    }
+
+    public static void setCurrentUserLikesPost(long postId, long blogId, boolean isLiked, long wpComUserId) {
```

Comment:
Overload the `setCurrentUserLikesPost()` function to handle the nullalbe `ReaderPost`